### PR TITLE
Add WebDriver conformance section

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

**Comment by nchevobbe** [comment]
@whimboo would you mind having a look?

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 1]
```diff
@@ -61,8 +61,17 @@ This article provides information about the changes in Firefox 112 that affect d
...
 
+- Implemented `browsingContext.print` as PDF command. See [Firefox bug 1806810](https://bugzil.la/1806810) for more details
+- Implemented `script.addPreloadScript` and `script.removePreloadScript` commands. See [Firefox bug 1806420](https://bugzil.la/1806420) and [Firefox bug 1806465](https://bugzil.la/1806465) for more details
+- Element and ShadowRoot references (NodeCache) now have the same `sharedId` in both Marionette and WebDriver BiDi, making it possible to use them in both protocol ([Firefox bug 1770733](https://bugzil.la/1770733)).
+- Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).
+
 #### Marionette
 
+- Fixed an issue happening on response wrapped in`value` field ([Firefox bug 1819029](https://bugzil.la/1819029)).
+-  Fixed an issue where `WebDriver:ElementClear` was emitting an extra `change` event for content editable elements ([Firefox bug 1744925](https://bugzil.la/1744925)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/112/index.md:72]:
```suggestion
- Fixed an issue where `WebDriver:ElementClear` was emitting an extra `change` event for content editable elements ([Firefox bug 1744925](https://bugzil.la/1744925)).
```

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 2]
```diff
@@ -61,8 +61,17 @@ This article provides information about the changes in Firefox 112 that affect d
...
+- Implemented `script.addPreloadScript` and `script.removePreloadScript` commands. See [Firefox bug 1806420](https://bugzil.la/1806420) and [Firefox bug 1806465](https://bugzil.la/1806465) for more details
+- Element and ShadowRoot references (NodeCache) now have the same `sharedId` in both Marionette and WebDriver BiDi, making it possible to use them in both protocol ([Firefox bug 1770733](https://bugzil.la/1770733)).
+- Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).
+
 #### Marionette
 
+- Fixed an issue happening on response wrapped in`value` field ([Firefox bug 1819029](https://bugzil.la/1819029)).
+- Fixed an issue where `WebDriver:ElementClear` was emitting an extra `change` event for content editable elements ([Firefox bug 1744925](https://bugzil.la/1744925)).
+
+
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/112/index.md:74]:
```suggestion
```

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 3]
```diff
@@ -61,8 +61,16 @@ This article provides information about the changes in Firefox 112 that affect d
 
 #### WebDriver BiDi
 
+- Implemented `browsingContext.print` as PDF command. See [Firefox bug 1806810](https://bugzil.la/1806810) for more details
+- Implemented `script.addPreloadScript` and `script.removePreloadScript` commands. See [Firefox bug 1806420](https://bugzil.la/1806420) and [Firefox bug 1806465](https://bugzil.la/1806465) for more details
+- Element and ShadowRoot references (NodeCache) now have the same `sharedId` in both Marionette and WebDriver BiDi, making it possible to use them in both protocol ([Firefox bug 1770733](https://bugzil.la/1770733)).
```
Comment (whimboo) [inline files/en-us/mozilla/firefox/releases/112/index.md:66]:
```suggestion
- `Element` and `ShadowRoot` references as stored in the node cache can now be used n both Marionette and WebDriver BiDi by their exact same unique reference. See [Firefox bug 1770733](https://bugzil.la/1770733) for more details.
```

**Review by whimboo — COMMENTED** [review]
FYI for all the WebDriver changes [see the bug list](https://bugzilla.mozilla.org/buglist.cgi?status_whiteboard=%5Bwebdriver%3Arelnote%5D&resolution=FIXED&query_format=advanced&v1=fixed&j_top=OR&status_whiteboard_type=substring&o1=equals&f1=cf_status_firefox112&list_id=16487821).

@nchevobbe thanks for the PR! Maybe we can get the additional information added as well?

File: files/en-us/mozilla/firefox/releases/112/index.md [hunk 4]
```diff
@@ -61,8 +61,16 @@ This article provides information about the changes in Firefox 112 that affect d
...
+- Removed `isRedirect` from the network events base parameters ([Firefox bug 1819875](https://bugzil.la/1819875)).
```
[the remark this answers is not in the document]
Comment (nchevobbe) [reply files/en-us/mozilla/firefox/releases/112/index.md:67]: I wanted to be a bit lighter for the simpler changes

[the remark this answers is not in the document]
Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/112/index.md:67]: If that's fine for the MDN content reviewers then it's fine for me as well.

**Review by whimboo — COMMENTED** [review]
Looks good to me. Just a small nit left to get fixed.

@teoli2003 feel free to merge when you are happy. Thanks!

**Review by dipikabh — APPROVED** [review]
👍

**Pull request merged**